net/macos-fortress/Portfile

```diff
@@ -3,8 +3,8 @@
 PortSystem          1.0
 
 name                macos-fortress
-version             2019.10.27
-revision            1
+version             2019.12.08
+revision            0
```

Comment:
This is a version update; the commit message makes it sound like it isn't.

Reply:
Thanks fixed. The mod started as a whitespace correction and grew…